routes/api.js

```diff
@@ -13,13 +13,16 @@ const badDomains = require('../bad-domains');
 const Twilio = require('../helpers/twilio');
 const moment = require('moment');
 const db = require('./../db/models');
+const utils = require('../helpers/utils');
 
 const { Sequelize } = db;
 const { Op } = Sequelize;
 
 const conveyorAccount = process.env.CONVEYOR_USERNAME;
 const conveyorKey = process.env.CONVEYOR_POSTING_WIF;
 
+const env = process.env.DATABASE_NAME;
```

Comment:
Let's call this something more obvious, like `DEBUG_MODE` or something. And do a logger.warn("IN DEBUG MODE") so we know when it is active

Reply:
Ok.